### PR TITLE
better docs

### PR DESCRIPTION
**Pull request by mamachanko**
* simplifies structure
* cleaner CLI docs

what's left
* further improve cli descriptions

### CODE REVIEW

File: docs/installation.rst [hunk 1]
```diff
@@ -1,46 +1,39 @@
...
-----------------------------------
+On Ubuntu
+---------
 
 .. code:: bash
 
 	$ sudo apt-get install build-essential python-dev python-pip
 
+.. FIXME is this necessary? ^
+
```
Comment (emulbreh) [inline docs/installation.rst]: Yes, it's necessary. But I agree that it should be mentioned somewhere else. We probably can assume that our target audience can install pip and python headers as needed.

File: docs/installation.rst [hunk 2]
```diff
@@ -48,24 +41,27 @@ Install and start ZooKeeper using:
...
 
 .. code:: bash
 
     $ brew install zookeeper
 
-You can edit the config file at ``/usr/local/etc/zookeeper/zoo.cfg``.
+ZooKeeper's configuration file is located at
+``/usr/local/etc/zookeeper/zoo.cfg``.
+
+.. FIXME how's RabbitMQ being installed?
```
Comment (emulbreh) [inline docs/installation.rst]: Mostly `brew install rabbitmq`

File: docs/todo [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+cli
+configuration
+testing
+lymph's protocol
+api reference
+faq
+contributing
+
+
+how to automatically document cli / docopt?
```
Comment (emulbreh) [inline docs/todo]: I don't think auto-generated docs would be helpful (you could just point to `lymph help` for that). The documentation should rather give usage examples and crossref related doc topics (like https://docs.djangoproject.com/en/1.8/ref/django-admin/)

Comment (emulbreh) [inline docs/todo]: Related note: why split the cli docs in multiple files?

File: docs/installation.rst (hunk 1 above)
Comment (mamachanko) [inline docs/installation.rst]: will include.

File: docs/installation.rst (hunk 2 above)
Comment (mamachanko) [inline docs/installation.rst]: yes. will do.

File: docs/todo [hunk 4]
```diff
@@ -0,0 +1 @@
+improve cli docstrings and include more details and examples
```
Comment (emulbreh) [inline docs/todo]: let's keep that in an issue instead

File: lymph/cli/config.py [hunk 5]
```diff
@@ -7,12 +7,13 @@ class ConfigCommand(Command):
...
 
-    Outputs the configuration.
+    Prints configuration for inspection
 
     {COMMON_OPTIONS}
+
     """
     
-    short_description = 'Show configuration.'
+    short_description = 'Prints configuration for inspection'
```
Comment (emulbreh) [inline lymph/cli/config.py]: Currently all descriptions end with a dot, we should probably unify by removing it everywhere.

File: docs/todo (hunk 4 above)
Comment (mamachanko) [inline docs/todo]: yes, sry. that was the idea.

File: lymph/cli/config.py (hunk 5 above)
Comment (mamachanko) [inline lymph/cli/config.py]: yes, had that at the back of my head. thanks.

File: docs/todo (hunk 4 above)
Comment (mamachanko) [inline docs/todo]: https://github.com/deliveryhero/lymph/issues/243

File: lymph/cli/config.py (hunk 5 above)
Comment (mamachanko) [inline lymph/cli/config.py]: done

File: docs/cli.rst [hunk 6]
```diff
@@ -1,184 +1,95 @@
...
-Prints a list of running services (and optionally instances).
 
-.. code::
+lymph discover
+----------------
 
-    $ lymph discover
-    echo [2]
-    demo [1]
+Discovers all available services and their instances, e.g.:
```
Comment (manisenkov) [inline docs/cli.rst]: Examples missed

Comment (mamachanko) [inline docs/cli.rst]: https://github.com/deliveryhero/lymph/issues/243

Comment (manisenkov) [inline docs/cli.rst]: Ah, ok

File: lymph/tests/integration/test_cli.py [hunk 7]
```diff
@@ -45,7 +45,7 @@ def test_negative_inspect(self):
 class ListCommandTests(CliTestMixin, unittest.TestCase):
     def test_list(self):
         self.assert_lines_equal(['list'], u"""
-            {t.bold}config     {t.normal}Show configuration.
+            {t.bold}config     {t.normal}Prints configuration for inspection
             {t.bold}tail       {t.normal}Stream the logs of one or more services.
```
Comment (manisenkov) [inline lymph/tests/integration/test_cli.py]: There can be used third-party form as well: "stream**s**", "send**s**"

**Comment by manisenkov** [comment]
:+1:

**Pull request merged**